src/joystick/hidapi/SDL_hidapijoystick.c

```diff
@@ -600,6 +732,26 @@ HIDAPI_JoystickInit(void)
         return 0;
     }
 
+#if defined(SDL_USE_LIBUDEV)
+    if (linux_enumeration_method == ENUMERATION_UNSET) {
+        if (SDL_getenv("SDL_HIDAPI_JOYSTICK_DISABLE_UDEV") != NULL) {
```

Comment:
Here it could be possible to reuse `SDL_JOYSTICK_DISABLE_UDEV`. I used a different environment variable to clearly separate the two, but in the end their job of disabling udev is very similar.
So probably even a single variable to control both should be acceptable.

Reply:
I'll defer to @slouken on that.